### PR TITLE
document slave pod image configuration env vars

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@gabemontero @bmcelvee ptal

File: using_images/other_images/jenkins.adoc [hunk 1]
```diff
@@ -277,6 +277,23 @@ config directory, this environment variable allows the fatal error log file to
 persist when a fatal error occurs. The fatal error file is saved at
 `/var/lib/jenkins/logs`.
 
+* `NODEJS_SLAVE_IMAGE`
++
+Setting this value will override the image used for the default nodejs agent pod configuration.
```
Comment (bmcelvee) [inline using_images/other_images/jenkins.adoc]: s/will override/overrides (in both paragraphs)

File: using_images/other_images/jenkins.adoc [hunk 2]
```diff
@@ -277,6 +277,23 @@ config directory, this environment variable allows the fatal error log file to
...
+The default nodejs agent pod uses `docker.io/openshift/jenkins-agent-nodejs-8-centos7` or
```
Comment (bmcelvee) [inline using_images/other_images/jenkins.adoc]: s/nodejs/NodeJS (in both paragraphs)

**Comment by bmcelvee** [comment]
Just a few things, otherwise LGTM

**Comment by bmcelvee** [comment]
Minor update, no QE required.

**Pull request merged**